diff -uNr yaml-cpp-0.5.1.orig/include/yaml-cpp/node/detail/iterator.h yaml-cpp-0.5.1/include/yaml-cpp/node/detail/iterator.h
--- yaml-cpp-0.5.1.orig/include/yaml-cpp/node/detail/iterator.h	2013-04-14 01:37:53.000000000 +0200
+++ yaml-cpp-0.5.1/include/yaml-cpp/node/detail/iterator.h	2019-07-24 15:23:47.333647517 +0200
@@ -11,6 +11,7 @@
 #include "yaml-cpp/node/detail/node_iterator.h"
 #include <boost/iterator/iterator_adaptor.hpp>
 #include <boost/utility.hpp>
+#include <boost/next_prior.hpp>
 
 namespace YAML
 {
